playwright-e2e/tests/dsm/participant-page/participant-page-name.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { expect, test } from '@playwright/test';
+import { login } from 'authentication/auth-dsm';
+import { study } from 'pages/dsm/navbar';
+import Select from 'lib/widget/select';
+import ParticipantsPage, { SearchFieldLabel } from 'pages/dsm/participants/participants-page';
+import Table from 'lib/widget/table';
+import { Page } from '@playwright/test';
+
+test.describe('Participant Page DSM', () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page);
+  });
+
+  test('Ensure name field updates properlyfor Brain @dsm @dsm-search', async ({ page }) => {
+    await nameTest("Brain", page);
+  });
+
+  test('Ensure name field updates properlyfor PanCan @dsm @dsm-search', async ({ page }) => {
+    await nameTest("PanCan", page);
+  });
+
+  test('Ensure name field updates properlyfor Singular @dsm @dsm-search', async ({ page }) => {
+    await nameTest("Singular", page);
+  });
+
+  test('Ensure name field updates properlyfor RGP @dsm @dsm-search', async ({ page }) => {
+    await nameTest("RGP", page);
+  });
+});
+
+
+async function nameTest(studyName: string, page: Page) {
+  await new Select(page, { label: 'Select study' }).selectOption(studyName);
+  await expect(page.locator('h1')).toHaveText('Welcome to the DDP Study Management System');
+  await expect(page.locator('h2')).toHaveText('You have selected the ' + studyName + ' study.');
+
+  await study(page).selectOption('Participant List', { waitForNav: true });
+  await expect(page.locator('h1')).toHaveText('Participant List', { timeout: 30 * 1000 });
+
+  const participantListPage = new ParticipantsPage(page);
+  await participantListPage.waitForReady();
+
+  //Pick the participant at row 4 and enter their participant page
+  const table = new Table(page);
+  const cell = await table.cell(3, 2)
+  const shortId = cell.innerText();
+  await cell.click();
+  await expect(page.locator('h1')).toHaveText('Participant Page', { timeout: 5 * 1000 });
+  //Change their first name
+  await page.fill('[placeholder="First Name"]', "test name");
```

Comment:
should we maybe change `test name` to something with a date time stamp so that later not all pts all called test name?
@aweng98 what naming convention do we currently use for naming pts

Reply:
+1
No naming convention yet. But please randomize names so it's different on each run. Maybe something in `faker-utils.ts` is suitable. Or https://github.com/faker-js/faker.

Reply:
I changed it to `John Wakeman-{currentDate}` Does this work?